USAGE.rst

```diff
@@ -39,26 +39,31 @@ To execute a build pipeline locally, try the following example command:
      python execute_pipeline.py \
        --pipeline_kwargs={\'sleep_secs\':2} SamplePipeline
 
-To execute a build pipeline remotely, start two consoles and run the following command
-in one console:
+Remote
+******
+
+To execute a build pipeline remotely, start two consoles (on the same or different machines) and run 
+the following command in one console:
```

Comment:
I know this isn't integral to this PR, but I'm confused about this - is this to run the pipeline on the console running the conductor, or to run the pipeline on a third machine in the Google infrastructure?